core/src/asu/gunma/ui/screen/game/GameScreen.java

```diff
@@ -127,16 +144,15 @@
 
     Preferences prefs;
 
-    public GameScreen(Game game, ActionResolver speechGDX, Music music, DbInterface dbCallback, Screen previous, ArrayList<VocabWord> activeList, Preferences prefs, GameAssets gameAssets) {
+    public GameScreen(Game game, ActionResolver speechGDX, Music music, DbInterface dbCallback, Screen previous, ArrayList<VocabWord> activeList, Preferences prefs) {
         this.game = game;
         this.prefs = prefs;
         this.speechGDX = speechGDX;
         this.dbCallback = dbCallback;
         this.previousScreen = previous;
         this.gameMusic = music;
         this.activeVList = activeList;
-        this.gameAssets = gameAssets;
-        gameMusic = Gdx.audio.newMusic(Gdx.files.internal(gameAssets.introMusicPath));
+        gameMusic = Gdx.audio.newMusic(Gdx.files.internal("IntroMusic.mp3"));
```

Comment:
same reason as above
```suggestion
        this.gameAssets = gameAssets;
        gameMusic = Gdx.audio.newMusic(gameAssets.introMusicPath);
```